Docs/Actions.md

```diff
@@ -14,6 +14,7 @@ In addition to the below short explanations, check out the inline documentation
 - `check-merge-conflict`: Labels and comments on pull requests with merge conflicts.
 - `create-jira-issues-for-community-activities`: Creates Jira issues for community activities happening on GitHub, like issues, discussions, and pull requests being opened. Pull requests are only taken into account if they're not already related to a Jira issue (by starting their title with a Jira issue key).
 - `enable-corepack`: Enables [Node corepack](https://nodejs.org/docs/latest-v16.x/api/corepack.html) so any package manager can be used seamlessly.
+- `login-to-azure`: Logs in to Azure using a service principal and the [azure/login action](https://github.com/Azure/login). 
```

Comment:
Similar to `checkout`, this should be an internally-only used action (there's no point in using it for others), without being advertised here. Also, similarly, add a not to the action itself.